src/core/jit/context.rs

```diff
@@ -1,22 +1,26 @@
-use async_graphql::{Name, SelectionField, ServerError};
+use async_graphql::{Name, ServerError};
 use async_graphql_value::ConstValue;
-use derive_getters::Getters;
 
-use super::Request;
-use crate::core::ir::ResolverContextLike;
+use super::{Field, Nested, Request};
+use crate::core::ir::{ResolverContextLike, SelectionField};
 
 /// Rust representation of the GraphQL context available in the DSL
-#[derive(Getters, Debug, Clone)]
+#[derive(Debug, Clone)]
 pub struct Context<'a, Input, Output> {
     request: &'a Request<Input>,
     value: Option<&'a Output>,
     args: Option<indexmap::IndexMap<Name, Input>>,
```

Comment:
in that case `args` becomes redundant since args are also present in field. But we can't just migrate it because it'll broke the some cases with default value. Maybe add todo to migrate?